run_tests.py

```diff
@@ -1,5 +1,6 @@
 
 import unittest
+import vcr
```

Comment:
unclear. I'm not exactly sure how python's importing works... possibly not. Totally defer to you.